src/System.CommandLine/Help/HelpBuilder.cs

```diff
@@ -17,11 +18,13 @@ public class HelpBuilder : IHelpBuilder
             = new Dictionary<ISymbol, Customization>();
 
         protected IConsole Console { get; }
+        protected Resources Resources { get; }
         public int MaxWidth { get; }
 
-        public HelpBuilder(IConsole console, int maxWidth = int.MaxValue)
+        public HelpBuilder(IConsole console, Resources? resources = null, int maxWidth = int.MaxValue)
         {
             Console = console ?? throw new ArgumentNullException(nameof(console));
+            Resources = resources ?? Resources.Instance;
```

Comment:
IMO rather than this being an option parameter, I would make it required, non-null, and throw on null. The fact that `maxWidth` is still an optional parameter is a bit of a hold-over from a previous version of the API.
```suggestion
        public HelpBuilder(IConsole console, Resources resources, int maxWidth = int.MaxValue)
        {
            Console = console ?? throw new ArgumentNullException(nameof(console));
            Resources = resources ?? throw new ArgumentNullException(nameof(resources));
```

Reply:
Resolved (7da17af1fbd8e26c12d652ac100ad383c4391e34).